src/modules/swap/containers/ConfirmSwapModalSetup/index.tsx

```diff
@@ -70,14 +57,6 @@ export function ConfirmSwapModalSetup(props: ConfirmSwapModalSetupProps) {
         onConfirm={handleSwap}
         onDismiss={handleConfirmDismiss}
       />
-
-      <TransactionConfirmationModal
-        attemptingTxn={true}
-        isOpen={showTransactionConfirmationModal}
-        pendingText={pendingText}
-        onDismiss={onDismiss}
-        operationType={operationType}
-      />
```

Comment:
This seemed redundant, as another instance is invoked from within `TransactionConfirmationModal`, so I just removed.
For what I tested (SWAP) it was not missed.

Reply:
True, also checked it.
Thanks for the cleaning up!

Reply:
@alfetopito wait, it seems to be used for Wrap/Unwrap flow

Reply:
I've refactored that part:
https://github.com/cowprotocol/cowswap/pull/2774
https://github.com/cowprotocol/cowswap/pull/2775

Reply:
Should I add it back?

Reply:
Yeah, there is no pending modal while Wrap/Unwrap